backend/server/tests/user/utility.py

```diff
@@ -14,3 +15,9 @@ def clear():
     """drop users in database. Used before every test is run."""
     setup_user_related_collections(drop=True)
     setup_redis_sessionsdb()
+
+def get_token():
+    return requests.post('http://127.0.0.1:8000/auth/guest_login').json()["session_token"]
+
+def get_token_headers():
+    return {"Authorization": f"Bearer {get_token()}"}
```

Comment:
highest key, this should take the token and not generate a new one itself. (and thus everywhere should first get_token and then get_token_headers with the previous token value)

Reply:
separation of responsibility